Library/Formula/tiff2png.rb

```diff
@@ -2,25 +2,15 @@
 
 class Tiff2png < Formula
   homepage 'http://www.libpng.org/pub/png/apps/tiff2png.html'
-  url 'ftp://ftp.simplesystems.org/pub/libpng/png/applications/tiff2png/tiff2png-0.91.tar.gz'
-  sha1 '3a23abaaadbed8f3d13b88241257fe2078eb61fd'
-  revision 1
+  url 'https://people.xiph.org/~giles/tiff2png/tiff2png-0.92.tar.gz'
+  sha256 'bf743ae5b231d690202eb8a552f173619815533172417aeb3f8a75e13315f0ed'
 
   depends_on 'libtiff'
   depends_on 'libpng'
   depends_on 'jpeg'
 
-  # libpng 1.5 no longer #includes zlib.h
-  patch :DATA
-
   def install
-    system "make", "-f", "Makefile.unx", "CC=#{ENV.cc}",
-                                         "OPTIMFLAGS=#{ENV.cflags}",
-                                         "LIBTIFF=#{HOMEBREW_PREFIX}/lib",
-                                         "TIFFINC=#{HOMEBREW_PREFIX}/include",
-                                         "LIBJPEG=#{HOMEBREW_PREFIX}/lib",
-                                         "ZLIB=/usr/lib",
-                                         "DEBUGFLAGS="
+    system "make"
     bin.install 'tiff2png'
```

Comment:
That doesn't create the target directory, so the copy fails:

/bin/cp -p tiff2png /usr/local/Cellar/tiff2png/0.92/bin/
cp: directory /usr/local/Cellar/tiff2png/0.92/bin does not exist